packages/containers/src/List/List.container.js

```diff
@@ -102,6 +102,13 @@ class List extends React.Component {
 			list: {
 				items,
 				columns: get(this.props, 'list.columns', []),
+				sort: {
+					field: state && state.sortOn,
+					isDescending: state && !state.sortAsc,
+					onChange: (event, data) => {
```

Comment:
`onChange: this.onSelectSortBy` should be sufficient, no need to create a new function

Reply:
:+1: